LocalSettings.php

```diff
@@ -160,6 +160,13 @@
 			'tag' => true,
 		),
 	),
+	'wgAbuseFilterBlockDuration' => array(
+		'default' => 'indefinite',
+	),
+	'wgAbuseFilterAnonBlockDuration' => array(
+		'default' => null
```

Comment:
Here. You can simply it by just making the default "6 months" as we really shouldn't be indefinitely blocking IPs